apps/chat/src/components/Chatbar/ChatbarSettings.tsx

```diff
@@ -99,6 +99,7 @@ export const ChatbarSettings = () => {
         Icon: IconSquareCheck,
         onClick: () => {
           dispatch(ConversationsActions.setAllChosenConversations());
+          dispatch(ConversationsActions.setAllChosenEmptyFolders());
```

Comment:
Why do we need separate action for that? We can do it into `setAllChosenConversations`

Reply:
I think it's better to implement separate actions for improved separation of concerns, testability, and reusability.

Reply:
Code chsnged as suggested.